Makefile

```diff
@@ -12,6 +12,11 @@ ALL_STACKS:=minimal-kernel \
 	datascience-notebook \
 	pyspark-notebook \
 	all-spark-notebook
+
+ALL_SINGLEUSERS:=$(shell echo $(ALL_STACKS) | sed "s/ /\n/g" | grep notebook | sed s/notebook/singleuser/g)
```

Comment:
echoing this gives me `minimal-kernelnminimal-singleusernr-singleusernscipy-singleuserndatascience-singleusernpyspark-singleusernall-spark-singleuser`.

Lack of whitespace is going to screw up later steps?